### DISCUSSION TITLE
airflow.api.auth.backend.session

### DISCUSSION THREAD

**Discussion by aamster** [apache/airflow#30754] [General]
### What do you see as an issue?

Unfortunately it's not described how to use the default authentication method

### Solving the problem

Please describe how to authenticate using the default authentication method

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
Authenticatioj is described there: https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/security/api.html#basic-authentication

IMHO is enough. I cannot find anything missing,. 

If you write here what you need and what you tried and what failed, there is a chance someone will help you and you will be able to update the documentation yourself after you get helped. Airlfow (including docs) is created by ~2500 contributors - people like you who attempt to use the free software and if they struggle, they actively seek for help desribing not general "I want the docs to be better" but describing what they've done and what did not work and what they tried and asking for help. Then often when they got help, they contributed back the documentaiton improving it with what they learned. 

Please explain what you've done to achieve what you wanted and ask for help and if you get it, it would be great if you contribute back the description of what you've learned while getting help. It's easy to contribute back "suggest a change on this page" button.

Converted it into a discussion and waiting for your struggle history, happy to help if you explain what you tried and what did not work and what you tried to achieve.
